status-loop/queryValidators.js

```diff
@@ -190,41 +191,53 @@ async function getEstimateInUsd(campaign) {
 async function queryValidators() {
 	const campaignsCol = db.getMongo().collection('campaigns')
 	const channels = await getChannels()
+	try {
```

Comment:
why is this wrapped in a try-catch

Reply:
I wanted to try-catch an error that was throwing there when I was testing. I can remove that as the error is not handled but just logged.